From 2809c0e8e602aa1a72967aa88c96009dc93651c2 Mon Sep 17 00:00:00 2001
From: Ronan Abhamon <ronan.abhamon@vates.fr>
Date: Mon, 12 Sep 2022 15:56:09 +0200
Subject: [PATCH 073/170] fix(linstorvolumemanager): remove double prefix on kv
 group name

- Before this patch, when the kv store was created/accessed, a double "xcp-sr-" prefix was used.
- This change is not compatible with existing LINSTOR SR instances!

Signed-off-by: Ronan Abhamon <ronan.abhamon@vates.fr>
---
 drivers/linstorvolumemanager.py | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/drivers/linstorvolumemanager.py b/drivers/linstorvolumemanager.py
index a1bc151..3ee5d24 100755
--- a/drivers/linstorvolumemanager.py
+++ b/drivers/linstorvolumemanager.py
@@ -2195,7 +2195,7 @@ class LinstorVolumeManager(object):
 
     def _create_linstor_kv(self, namespace):
         return linstor.KV(
-            self._get_store_name(),
+            self._group_name,
             uri=self._linstor.controller_host(),
             namespace=namespace
         )
@@ -2205,9 +2205,6 @@ class LinstorVolumeManager(object):
         properties.namespace = self._build_volume_namespace(volume_uuid)
         return properties
 
-    def _get_store_name(self):
-        return 'xcp-sr-{}'.format(self._group_name)
-
     @classmethod
     def _build_sr_namespace(cls):
         return '/{}/'.format(cls.NAMESPACE_SR)
-- 
2.44.0

